roles/ovirt-cluster-upgrade/README.md

```diff
@@ -0,0 +1,57 @@
+oVirt cluster upgrade
+=========
+
+This role will iterate through all hosts in cluster and upgrades them.
+
+Requirements
+------------
+
+ * Ansible version 2.3 or higher
+ * Python SDK version 4 or higher
+
+Role Variables
+--------------
+
+| Name               | Default value         |                            |
+|--------------------|-----------------------|----------------------------| 
+| cluster            | Default               | Name of the cluster to be upgraded |
+| low_prio_vms       | UNDEF                 | List of VM names to be stopped before upgrade |
+| force_pinned       | false                 | If true the pinned to host VMs will be stopped |
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+```yaml
+---
+- name: oVirt infra
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://ovirt-engine.example.com/ovirt-engine/api
+    engine_user: admin@internal
+    engine_password: 123456
+    engine_cafile: /home/omachace/ovirt-engine41/etc/pki/ovirt-engine/ca.pem
+
+    cluster_name: production
+    low_prio_vms:
+      - openshift_master_0
+      - openshift_node_0
+      - openshift-node-image
```

Comment:
Most of their roles are not documented. Also most of their roles are different, it's mostly provisioning of hosts, those are mostly buidling of infra. But if I take an example of firewalld they use name like httpd, and specific number, instead of `myservice` and port `1234`.